.github/workflows/rubocop.yaml

```
@@ -0,0 +1,27 @@
+name: Rubocop
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.7
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+    - name: Cache gems
+      uses: actions/cache@v1
```

```suggestion
      uses: actions/cache@v2
```
V1 won't work for us (with the pattern `restore-keys`)